pkg/filter/kafka/spec.go

```diff
@@ -21,5 +21,13 @@ type (
 	// Spec is spec of Kafka
 	Spec struct {
 		Backend []string `yaml:"backend" jsonschema:"required,uniqueItems=true"`
+		KVMap   *KVMap   `yaml:"mqtt" jsonschema:"required"`
```

Comment:
Kafka spec could contain topic (string) and headers (map[string]string) directly. It no need to introduce a sub struct `KVMap`

Reply:
`KVMap` defines keys to get values from MQTT context KVMap. For example, if set KVMap.Topic = "kafkaTopic", then topic for kafka backend will be MQTTContext.GetKV("kafkaTopic"). maybe i use wrong name.